src/styles/audit/audit.css

```diff
@@ -0,0 +1,143 @@
+.audit {
+  padding-top: 50px;
+}
+
+.all-time-investment {
+  width: 48%;
+}
+
+.all-time-disbursement {
+  width: 48%;
+}
+
+.all-time-investment .total-summary {
+  padding-top: 20px;
+  font-size: 12px;
+}
+
+.all-time-disbursement .total-summary {
+  padding-top: 20px;
+  font-size: 12px;
+}
+
+.audit-filter {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 15px;
+  padding-bottom: 20px;
+}
+
+.filter-name {
+  padding-left: 5px;
+}
+
+.audit-transactions {
+  padding-top: 20px;
+}
+
+.audit-transactions .transaction-date {
+  color: rgb(90, 90, 90);
+}
+
+.audit-transactions .name {
+  font-weight: bold;
+  font-size: 14px;
+}
+
+.audit-transactions .type {
+  text-align: left;
+}
+
+.audit-transactions .amount {
+  color: rgb(18, 61, 18);
+}
+
+.audit-item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.5rem 1.8rem;
+  border-radius: 5px;
+  border: 1px solid #e3e3e3;
+  font-size: 0.85rem;
+  color: #5e5e5e;
+  margin-bottom: 1.2rem;
+}
+
+.audit-item .name {
+  flex: 1;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.audit-item .c {
+  width: 20%;
+  text-align: center;
+}
+
+.audit-item .amt {
+  color: #05922d;
+  flex: 1;
+  width: 30%;
+  white-space: nowrap;
+  text-align: right;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+@media screen and (max-width: 520px) {
+  .audit-item {
+    font-size: 13px;
+  }
+
+  .audit-item .c {
+    margin-left: 20px;
+    width: 20%;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+}
+
+@media screen and (max-width: 769px) {
+  .total-sum-card {
+    flex-direction: column !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5561530)